barretenberg/cpp/src/barretenberg/sumcheck/sumcheck_round.hpp

```diff
@@ -507,14 +507,24 @@ template <typename Flavor> class SumcheckProverRound {
 
         // In Round 0, we have to compute the contribution from 2 edges: (1, 1,..., 1) and (0, 1, ..., 1) (as points on
         // (d-1) - dimensional Boolean hypercube).
-        size_t start_edge_idx = (round_idx == 0) ? round_size - 4 : round_size - 2;
+        size_t start_edge_idx{ 0 };
```

Comment:
Minor comment: I guess comment above should now say except if we're ultra zk flavor or something like that?